bin/handler-opsgenie.rb

```diff
@@ -54,7 +54,7 @@ def process
       when '200', '202'
         puts 'opsgenie -- ' + @event['action'].capitalize + 'd incident -- ' + event_id
       when '401'
-        puts 'opsgenie -- failed to ' + @event['action'] + ' incident -- not authorized'
+        puts 'opsgenie -- failed to ' + @event['action'] + ' incident -- ' + event_id + ': not authorized'
```

Comment:
This change of failure message makes things _a)_ more consistent, and _b)_ allows for the negative tests to be used for detecting the event ID generated for an alert.

Reply:
nice, I am thinking that rather than doing string concatenation we might want to just opt for the built in string interpolation. While this would not have a significant performance gain in this example it is generally faster to use string interpolation over concatenation. IMHO the biggest reason to use string interpolation here is because it is way easier to read.

It would look something like this:
```ruby
puts "opsgenie -- failed to #{@event['action']} incident -- #{event_id}: not authorized"
```

Reply:
Done.